setup.py

```diff
@@ -232,6 +233,7 @@ def run(self):
 # when modifying the following list, make sure to update src/transformers/dependency_versions_check.py
 install_requires = [
     deps["dataclasses"] + ";python_version<'3.7'",  # dataclasses for Python versions that don't have it
+    deps["importlib_metadata"] + ";python_version<'3.8'",  # importlib_metadata for Python versions that don't have it
```

Comment:
This new dependency is for python versions < 3.8 (a bit like dataclasses). The package allows us to inspect the metadata of a library without importing it.